[2020-12-22 13:58:54,027] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: loan_prediction_train_model.preprocess_data 2020-12-22T12:43:23.210460+00:00 [queued]>
[2020-12-22 13:58:54,042] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: loan_prediction_train_model.preprocess_data 2020-12-22T12:43:23.210460+00:00 [queued]>
[2020-12-22 13:58:54,042] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-12-22 13:58:54,042] {taskinstance.py:867} INFO - Starting attempt 33 of 36
[2020-12-22 13:58:54,043] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-12-22 13:58:54,058] {taskinstance.py:887} INFO - Executing <Task(LivyBatchOperator): preprocess_data> on 2020-12-22T12:43:23.210460+00:00
[2020-12-22 13:58:54,063] {standard_task_runner.py:53} INFO - Started process 53499 to run task
[2020-12-22 13:58:54,145] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: loan_prediction_train_model.preprocess_data 2020-12-22T12:43:23.210460+00:00 [running]> dcb43f18a24b
[2020-12-22 13:58:54,181] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,180] {batch.py:214} INFO - Submitting the batch to Livy... Payload:
{
  "file": "s3a://alanchn31-loan-default-prediction/dist/main.py",
  "args": [
    "--job",
    "preprocess_data",
    "--phase",
    "train",
    "--mode",
    "aws",
    "--awsKey",
    "AKIAUW4FKHTMR2UR3INH",
    "--awsSecretKey",
    "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",
    "--s3Bucket",
    "alanchn31-loan-default-prediction"
  ],
  "pyFiles": [
    "s3a://alanchn31-loan-default-prediction/dist/src.zip"
  ],
  "name": "preprocess_data_scheduled__2020-12-22T12:43:23.210460+00:00"
}
[2020-12-22 13:58:54,191] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,190] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:58:54,194] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,194] {http_hook.py:136} INFO - Sending 'POST' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches
[2020-12-22 13:58:54,224] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,223] {batch.py:166} INFO - Batch successfully submitted with id = 29.
[2020-12-22 13:58:54,226] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,226] {batch.py:80} INFO - Getting batch 29 status...
[2020-12-22 13:58:54,235] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,235] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:58:54,238] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,238] {http_hook.py:136} INFO - Sending 'GET' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/29
[2020-12-22 13:58:54,248] {logging_mixin.py:112} INFO - [2020-12-22 13:58:54,248] {batch.py:90} INFO - Batch 29 has not finished yet (state is 'starting')
[2020-12-22 13:59:14,270] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,270] {batch.py:80} INFO - Getting batch 29 status...
[2020-12-22 13:59:14,278] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,278] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:59:14,281] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,280] {http_hook.py:136} INFO - Sending 'GET' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/29
[2020-12-22 13:59:14,289] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,289] {batch.py:299} INFO - --------------------------------------------------Full log for batch 29--------------------------------------------------
[2020-12-22 13:59:14,297] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,296] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:59:14,299] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,299] {http_hook.py:136} INFO - Sending 'GET' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/29/log?from=0&size=100
[2020-12-22 13:59:14,307] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,306] {batch.py:309} INFO - stdout: 
[2020-12-22 13:59:14,307] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,307] {batch.py:309} INFO - 2020-12-22 13:58:55,936 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2020-12-22 13:59:14,308] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,307] {batch.py:309} INFO - Traceback (most recent call last):
[2020-12-22 13:59:14,308] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,308] {batch.py:309} INFO -   File "/tmp/spark-a5b3aab1-4c24-4da2-8622-29f055b69357/main.py", line 74, in <module>
[2020-12-22 13:59:14,308] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,308] {batch.py:309} INFO -     main()
[2020-12-22 13:59:14,308] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,308] {batch.py:309} INFO -   File "/tmp/spark-a5b3aab1-4c24-4da2-8622-29f055b69357/main.py", line 55, in main
[2020-12-22 13:59:14,308] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,308] {batch.py:309} INFO -     with open("src/config.json", "r") as config_file:
[2020-12-22 13:59:14,309] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,309] {batch.py:309} INFO - FileNotFoundError: [Errno 2] No such file or directory: 'src/config.json'
[2020-12-22 13:59:14,309] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,309] {batch.py:309} INFO - 2020-12-22 13:58:59,476 INFO util.ShutdownHookManager: Shutdown hook called
[2020-12-22 13:59:14,309] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,309] {batch.py:309} INFO - 2020-12-22 13:58:59,477 INFO util.ShutdownHookManager: Deleting directory /tmp/spark-a5b3aab1-4c24-4da2-8622-29f055b69357
[2020-12-22 13:59:14,310] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,309] {batch.py:309} INFO - 2020-12-22 13:58:59,483 INFO impl.MetricsSystemImpl: Stopping s3a-file-system metrics system...
[2020-12-22 13:59:14,310] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,310] {batch.py:309} INFO - 2020-12-22 13:58:59,483 INFO impl.MetricsSystemImpl: s3a-file-system metrics system stopped.
[2020-12-22 13:59:14,310] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,310] {batch.py:309} INFO - 2020-12-22 13:58:59,484 INFO impl.MetricsSystemImpl: s3a-file-system metrics system shutdown complete.
[2020-12-22 13:59:14,311] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,310] {batch.py:309} INFO - 
stderr: 
[2020-12-22 13:59:14,311] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,311] {batch.py:309} INFO - 
YARN Diagnostics: 
[2020-12-22 13:59:14,311] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,311] {batch.py:309} INFO - spark-submit start failed
[2020-12-22 13:59:14,311] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,311] {batch.py:318} INFO - --------------------------------------------------End of full log for batch 29--------------------------------------------------
[2020-12-22 13:59:14,312] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,312] {batch.py:335} INFO - Closing batch with id = 29
[2020-12-22 13:59:14,329] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,329] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:59:14,330] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,330] {http_hook.py:136} INFO - Sending 'DELETE' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/29
[2020-12-22 13:59:14,346] {logging_mixin.py:112} INFO - [2020-12-22 13:59:14,346] {batch.py:340} INFO - Batch 29 has been closed
[2020-12-22 13:59:14,357] {taskinstance.py:1128} ERROR - Batch 29 failed with state 'dead'
Traceback (most recent call last):
  File "/opt/bitnami/python/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/airflow/plugins/airflow_livy/batch.py", line 173, in execute
    ).execute(context)
  File "/opt/bitnami/python/lib/python3.6/site-packages/airflow/sensors/base_sensor_operator.py", line 107, in execute
    while not self.poke(context):
  File "/usr/local/airflow/plugins/airflow_livy/batch.py", line 96, in poke
    raise AirflowException(f"Batch {self.batch_id} failed with state '{state}'")
airflow.exceptions.AirflowException: Batch 29 failed with state 'dead'
[2020-12-22 13:59:14,359] {taskinstance.py:1151} INFO - Marking task as UP_FOR_RETRY
[2020-12-22 13:59:19,071] {logging_mixin.py:112} INFO - [2020-12-22 13:59:19,071] {local_task_job.py:103} INFO - Task exited with return code 1
